Sources/RepoToolsCore/RepoActions.swift

```diff
@@ -407,7 +407,7 @@ public enum RepoActions {
                         fileName: escape(download)
                     ),
                 ])
-            } else if lowercasedFileName.hasSuffix("tar.gz") {
+            } else if lowercasedFileName.hasSuffix("tar.gz") || lowercasedFileName.hasSuffix("tgz") {
                 return shell.command(CommandBinary.sh, arguments: [
                     "-c",
                     untarTransaction(
```

Comment:
(GitHub won't let me comment on the line I actually care about, which is below) While we're here we might want to update the message below to ...`other than .zip, .tar.gz, or .tgz`

Reply:
Fixed: https://github.com/discordapp/PodToBUILD/pull/1/commits/ebeffa42837078a873fcf48ad0b01fc20aa01bb6#diff-8899a6f8f09176941c964bb6deee3f2cR440